docker/volumes/db/webhooks.sql

```diff
@@ -205,4 +205,116 @@ BEGIN;
   ALTER function supabase_functions.http_request() SET search_path = supabase_functions;
   REVOKE ALL ON FUNCTION supabase_functions.http_request() FROM PUBLIC;
   GRANT EXECUTE ON FUNCTION supabase_functions.http_request() TO postgres, anon, authenticated, service_role;
+  INSERT INTO supabase_functions.migrations (version) VALUES ('20240115163000_add_retry_to_http_request');
+  CREATE OR REPLACE FUNCTION supabase_functions.http_request()
```

Comment:
It would be nice to update this function in place on line 28. This file is only run on fresh start of database so we don't need to worry about backwards compatibility with existing migrations.